contracts/container/config.yml

```diff
@@ -66,7 +67,6 @@ events:
       - name: LimitValue
         type: Integer
       - name: Hard
```

Comment:
Name with no type?

Reply:
cut/paste problems, why did it succeed in compiling?